### PR TITLE
fix windows ninja build failed

### PR DESCRIPTION
**Pull request by Hlongyu**
https://github.com/lfreist/hwinfo/issues/102

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -104,14 +104,14 @@ if (HWINFO_DISK)
     add_library(hwinfo_disk_static STATIC ${DISK_SRC_FILES})
     target_include_directories(hwinfo_disk_static PUBLIC $<BUILD_INTERFACE:${HWINFO_INCLUDE_DIR}> $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
     if(APPLE)
-        target_link_libraries(hwinfo_disk_static PRIVATE "-framework IOKit" "-framework CoreFoundation")
+        target_link_libraries(hwinfo_disk_static PRIVATE "-framework IOKit" "-framework CoreFoundation_static")
```
Comment (lfreist) [inline src/CMakeLists.txt:107]: CoreFoundation is an OSX framework, there is no CoreFoundation_static available as far as I know...

**Comment by lfreist** [comment]
Do you know, why the static postfix is necessary on windows? Shouldn't the .dll/.lib file extensions be sufficient?

**Comment by Hlongyu** [comment]
> Do you know, why the static postfix is necessary on windows? Shouldn't the .dll/.lib file extensions be sufficient to indicate static and dynamic builds?

On the Windows platform, it's true that 'static' is not needed in the naming of static libraries. However, I noticed that Ninja seems to compile both dynamic and static libraries simultaneously, and I misunderstood the meaning of OUTPUT_NAME. Adding '_static' was a mistake; I'll fix it tomorrow.

**Comment by Hlongyu** [comment]
I think we need an option to use shared or static.  
Like https://github.com/recp/cglm/blob/48839a38a14a653c233693fa911348ce58d57167/CMakeLists.txt#L14

**Comment by lfreist** [comment]
Thank you! One last thing: The system_info_standalone target must be removed from the tests.

**Review by lfreist — APPROVED** [review]

**Pull request merged**